tornjak-frontend/Utils/index.js

```diff
@@ -0,0 +1,25 @@
+import checkPropTypes from "check-prop-types";
```

Comment:
Why do we have 2 `Utils/` directory, one in `tornjak-frontend/Utils` and one `tornjak-frontend/src/Utils`. Can they be combined?